code-challenges.test.js

```diff
@@ -1,59 +1,163 @@
+
+
+
+
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
+
+// a) Create a test with expect statements for each of the variables provided.
+
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
+
+// b) Create the function that makes the test pass.
+
+// --------------------1) Create a function that takes an object as an argument and decides if the number inside it is evenly divisible by three or not.
+
+// a) Create a test with expect statements for each of the variables provided.
+
+// input: an object containing a number
+// output: a string stating wether or not the number inside is evenly divisible by three
+
+// Pseudocode: Create a function that will hold an if/else statement that will take an input and check if it is a number that is evenly divisible by 3. If it is, return a string stating that `${whateverTheInputIs} is divisible by three`. if it is a number that is NOT evenly divisible by 3, return a string stating `${whateverTheInputIs} is not divisible by three`. if the input is not a number, return an error message stating such.
+
+describe("cerberus", () => {
+    const object1 = { number: 15 }
+    const object2 = { number: 0 }
+    const object3 = { number: -7 }
+    it("checks if a number is evenly divisible by three", () => {
+        expect(cerberus(object1.number)).toEqual(`${object1.number} is divisible by three! :D`)
+        expect(cerberus(object2.number)).toEqual(`${object2.number} is divisible by three! :D`)
+        expect(cerberus(object3.number)).toEqual(`${object3.number} is not divisible by three! >:(`)
+    })
+});
+// [1st error] ReferenceError: divDecider is not defined
+// [2nd error]  Expected: "[object Object] is divisible by three!" Received: undefined  /// ----> didn't give the test any values to work with.
+
+
+
+// Expected output: "15 is divisible by three"
+// Expected output: "0 is divisible by three"
+// Expected output: "-7 is not divisible by three"
+
+// b) Create the function that makes the test pass.
+
+
+const cerberus = (input) => {     // *****I spent infinitely too long refactoring this question in order to stop it working on boolean values and strings, while still accepting either numbers or objects with the 'number:' property...*****

```

Comment:
This is an awesome stretch! It literally makes me so happy to see you get this kind of thing working!